src/bootctl/bootctl.c

```diff
@@ -237,6 +237,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "\n%3$sBoot Loader Interface Commands:%4$s\n"
                "  set-default ID       Set default boot loader entry\n"
                "  set-oneshot ID       Set default boot loader entry, for next boot only\n"
+               "  set-sysfail ID       Set boot loader entry to be used in case of a sys failure\n"
```

Comment:
in the English explanatory string please don't abbreviate "system" unnecessarily, i.e. on the right hand side please write out "system failure"